monosi/anomalies/__init__.py

```diff
@@ -28,12 +28,9 @@ def _std_dev(cls, values: List[float]):
 
     @classmethod
     def _z_scores(cls, stats: List[MetricStat], filter_none=True):
-        stats_to_calc = stats
-
         if filter_none:
-            stats_to_calc = [stat for stat in stats if stat.value != None]
-
-        values = [stat.value for stat in stats_to_calc]
+            stats = list(filter(lambda x: x.value, stats))
```

Comment:
Tried this out in the python shell, looks like it filters out the 0's as well, we just want the `NoneType` filtered 
![image](https://user-images.githubusercontent.com/14094284/148653102-919e0f3f-a1cb-4b5d-9b7d-782c8a0da3d5.png)

Reply:
Amazing catch - thank you